pandas/core/format.py

```diff
@@ -2106,7 +2106,9 @@ def _format_strings(self):
 class PeriodArrayFormatter(IntArrayFormatter):
 
     def _format_strings(self):
-        values = np.array(self.values.to_native_types(), dtype=object)
+        values = self.values
```

Comment:
In Series, ``self.values`` is already ``np.ndarray`` of period. No need to do round-trip?